src/gax-internal/src/grpc.rs

```diff
@@ -232,14 +220,18 @@ impl Client {
         Ok(headers)
     }
 
-    fn get_retry_policy(
-        &self,
-        options: &gax::options::RequestOptions,
-    ) -> Option<Arc<dyn RetryPolicy>> {
+    fn get_retry_policy(&self, options: &gax::options::RequestOptions) -> Arc<dyn RetryPolicy> {
         options
             .retry_policy()
             .clone()
             .or_else(|| self.retry_policy.clone())
+            .unwrap_or_else(|| {
+                Arc::new(
+                    Aip194Strict
+                        .with_attempt_limit(10)
+                        .with_time_limit(Duration::from_secs(5 * 60)),
```

Comment:
I suggest we use a lower value, maybe 60s.  I got complaints about storage being "stuck" with timeouts in that range.  But I am speculating about what will trigger fewer complaints in corner cases, most of the time 60s or 300s will behave exactly the same.

Reply:
Done.